test/system/090-events.bats

```diff
@@ -25,3 +25,35 @@ load helpers
     run_podman events --filter type=container --filter container=$cname --filter event=start --stream=false
     is "$output" "$expect" "filtering just by label"
 }
+
+@test "image events" {
+    # `remove events` seem to be broken...
+    skip_if_remote "FIXME: -remote is missing many events"
+    pushedDir=$PODMAN_TMPDIR/dir
+    mkdir -p $pushedDir
+
+    tarball=$PODMAN_TMPDIR/ball.tar
+
+    run_podman image inspect --format "{{.ID}}" $IMAGE
+    imageID="$output"
+
+    tag="registry.com/image:newtag"
```

Comment:
I'd really like to see some use of `random_string` here (probably with `| tr A-Z a-z` to ensure lower-case. Some day I should fix random_string so it does that automatically). Reason: with a fixed name, we could be seeing results of a prior test run.

Reply:
Will keep that in mind for the future. Thanks, Ed!